docs/messageid.md

```diff
@@ -0,0 +1,27 @@
+# Key Vault plugin for Azure Service Bus
```

Comment:
Update the description as well

---
In reply to: [120996416](https://github.com/Azure/azure-service-bus-dotnet-plugins/pull/5#discussion_r120996416) [](ancestors = 120996416)